client/storage-monitor/src/lib.rs

```diff
@@ -117,10 +117,10 @@ impl StorageMonitorService {
 
 	/// Returns free space in MB, or error if statvfs failed.
 	fn free_space(path: &Path) -> Result<u64, Error> {
-		Ok(fs4::available_space(path).map(|s| s / 1_000_000)?)
+		Ok(fs4::available_space(path).map(|s| s / 1024 / 1024)?)
```

Comment:
I think that the returned value was purposely returned in MB and not in MiB.
But maybe @michalkucharczyk knows better

Reply:
Well I made this decision basing on https://en.wikipedia.org/wiki/Megabyte being `10^6` not `1024^2`.
Feel free to change it if you really feel like we need to use `MiB` (Mebibyte). But please also adjust `clap` `args` documentation.

Reply:
If we return the value in MiB then also change the docs (MB -> MiB) please

Reply:
Well, I think we should use `MiB`(1024 * 1024 Byte). 
This is more accurate for node consumption of storage resources.

Reply:
ok, for me. but please update docs/comments wherever needed.